### ISSUE TITLE
Hey @MohammedEssehemy ! I tried to use verify-pdf but when I ran it over a pdf with two signatures I get this output:

### ISSUE THREAD

**Issue by MohammedEssehemy** [ninja-labs-tech/verify-pdf#8]
Hey @MohammedEssehemy ! I tried to use verify-pdf but when I ran it over a pdf with two signatures I get this output:
``` 
    console.log src/signpdf.test.js:162
      { verified: false,
        authenticity: false,
        integrity: true,
        expired: false,
        meta:
         { certs: [ [Object] ],
           signatureMeta:
            { reason: 'first',
              contactInfo: 'emailfromp1289@gmail.com',
              location: 'Location from p12' } } }
```
It doesn't seem to recognize the second signature.
If I run mutools I get this output:
```
➜  node-signpdf git:(develop) ✗ mutool sign -v resources/signed-twice.pdf                   
verifying signature 18
  Designated name: CN=signpdf, O=Some Company, OU=, emailAddress=signpdf@example.com, C=BG
  Certificate error: Self-signed certificate.
  The signature is valid but there have been edits since signing.
verifying signature 21
  Designated name: CN=SignPdf, O=node-signpdf, OU=, emailAddress=node-signpdf@example.com, C=BG
  Certificate error: Self-signed certificate.
  The document is unchanged since signing.
```
The file is this:
[signed-twice.pdf](https://github.com/vbuch/node-signpdf/files/3810645/signed-twice.pdf)

_Originally posted by @andres-blanco in https://github.com/vbuch/node-signpdf/pull/38#discussion_r340577109

**Comment by gribera** [comment]
Hi.. I want to collaborate to finish this feature.. what would be the expected output? an array with all the signatures instead of an object like now?

**Comment by MohammedEssehemy** [comment]
@gribera sure, go forward and work on it

**Comment by mohamedgomran** [comment]
@MohammedEssehemy Let's close this issue, fixed in [#113 ]
